app/main.go

```diff
@@ -74,16 +75,24 @@ func main() {
 }
 
 func setup(o opts) (n notifier, p publisher.Interface, err error) {
+	content, err := ioutil.ReadFile("exclusion-patterns.txt")
+	if err != nil {
+		log.Print("[WARN] could not read 'exclusion-patterns.txt' file")
+	}
+	lines := strings.Split(string(content), "\n")
```

Comment:
this is unsafe. in case of err we don't want to touch the content.

Reply:
Ok, I'm initializing content to an empty slice of bytes now